front/app/containers/ProjectsShowPage/timeline/VotingResults/VotingResultCard.tsx

```diff
@@ -224,7 +224,7 @@ const VotingResultCard = ({ idea, phaseId, rank }: Props) => {
         </Header>
 
         <Body>
-          <Box h="100%" display="flex" alignItems="flex-end">
+          <Box h="100%" alignContent="flex-end">
             <ProgressBar idea={idea} phase={phase} />
           </Box>
```

Comment:
This fixes the issue where the vote bars were really short (not taking up full width of card). I didn't see any negative impact on other card types when making this change.